tests/search/partialupdate/res_hnsw_1.json

```diff
@@ -0,0 +1,8 @@
+{"root":{"id":"toplevel","relevance":1.0,"fields":{"totalCount":1},
+	"coverage":{"coverage":100,"documents":1,"full":true,"nodes":1,"results":1,"resultsFull":1},
+	"children":[{"id":"id:foobar:hnsw_search::1","relevance":0.40408804594475856,"source":"search",
```

Comment:
Consider adding a default rank-profile with score 0 to avoid the dependency to nativeRank.